vendor/github.com/BurntSushi/toml

```diff
@@ -0,0 +1 @@
+Subproject commit f0aeabca5a127c4078abb8c8d64298b147264b55
```

Comment:
deps.sh creates a detached branch based on HEAD when it first adds the submodule.